docs/reference/scripting/using.asciidoc

```diff
@@ -197,6 +197,86 @@ them in other cases to remove ambiguity.
 Use this abbreviated syntax anywhere that {es} supports scripts, such as
 when you're creating <<runtime-mapping-fields,runtime fields>>.
 
+[discrete]
+[[script-fields-api]]
+=== Access document fields
+The examples from previous sections use the `field` API to access document
+fields:
+
+[source,painless]
+----
+field('my_field').get(null)
```

Comment:
Let's use a different default than null here, empty string or 1.

Reply:
Since we're describing the basic syntax, let's just use an empty string for `get()`.